community/modules/file-system/DDN-EXAScaler/outputs.tf

```diff
@@ -24,16 +24,47 @@ output "ssh_console" {
   value       = module.ddn_exascaler.ssh_console
 }
 
+output "client_config" {
+  description = "Script that will install DDN EXAScaler lustre client. The machine running this script must be on the same network & subnet as the EXAScaler."
```

Comment:
For my own knowledge, why is it that the machine must be on the same network/subnet?

Reply:
The script downloads a binary from the ddn server. Then invokes the binary. I know the binary does something with firewall rules, in addition to installing the client. So something to do with all that although I don't know in detail. The requirement is documented by DDN [here](https://github.com/DDNStorage/exascaler-cloud-terraform/tree/master/gcp#install-new-exascaler-cloud-clients).